test/celeritas/em/UrbanMsc.test.cc

```diff
@@ -203,6 +204,8 @@ class UrbanMscTest : public GlobalGeoTestBase
     RandomEngine                       rng_;
 
     std::shared_ptr<UrbanMscModel> model_;
+
+    detail::ScopedRootErrorHandler scoped_root_error_;
```

Comment:
I don't think the UrbanMsc test uses ROOT data? It might also be easier to put the `ScopedRootErrorHandler` in the `test/testdetail/TestMainImpl.cc` file, though that would necessitate adding the `celeritas` library target to `testcel_harness`.

Reply:
It does use the `RootImporter`, doesn't it?

Reply:
Yep quite so. Perhaps scope it to the setup function?

Reply:
done